src/components/events/EventForm.jsx

```diff
@@ -63,6 +70,12 @@ class EventForm extends React.Component {
     onModalClose();
   }
 
+  //Format of date must be DD/MM/YYYY
```

Comment:
When we do these function comments, could we phrase all statements in full sentences?